platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ElectionRound.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.consensus;
+
+import static com.swirlds.logging.LogMarker.CONSENSUS_VOTING;
+
+import com.swirlds.common.utility.IntReference;
+import com.swirlds.platform.Utilities;
+import com.swirlds.platform.event.EventConstants;
+import com.swirlds.platform.event.EventMetadata;
+import com.swirlds.platform.internal.EventImpl;
+import com.swirlds.platform.state.MinGenInfo;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * A round whose witnesses are currently having their fame voted on in elections. This class tracks
+ * the witnesses and their decided status.
+ */
+public class ElectionRound {
+    private static final Logger logger = LogManager.getLogger(ElectionRound.class);
+    /** the round number of witnesses we are voting on */
```

Comment:
```suggestion
    /** the round created number of witnesses we are voting on */
```

Reply:
im not sure this is accurate, we should discuss